### DISCUSSION TITLE
二维数组列表自定义组件的实现问题

### DISCUSSION THREAD

**Discussion by daijingfeng** [alibaba/formily#3582] [Q&A 2.x]
[codesandbox链接](https://codesandbox.io/s/boring-monad-jbh6cv?file=/components/collapse-items.tsx)

<img height="500px" src="https://user-images.githubusercontent.com/17313078/204699454-99f0b585-bd27-4453-95a9-0f88f12d9119.png" />

业务诉求是实线如图的组件，<b>有N页，每1页里面有N项</b>，是一个二维数组的渲染。

现在的问题是，组件界面是按预期显示，但填写表单后没有得到填写的数据。

期望输出的数据结构
```javascript
{
  items: [
    {
      items: [
        { input: "1", select: 1 },
        { input: "2", select: 2 },
        { input: "3", select: 3 }
      ]
    },
    {
      items: [
        { input: "4", select: 1 },
        { input: "5", select: 2 },
        { input: "6", select: 3 }
      ]
    }
  ]
}
```

实际输出的结构
```javascript
{
  items: [
    {},
    {},
  ]
}
```

麻烦大佬帮忙看看。

**Comment by janryWang** [comment]
自己在提交的时候做转换
